ansible.cfg

```diff
@@ -26,6 +33,12 @@ nocows = True
 # Uncomment to use the provided AWS dynamic inventory script
 #hostfile = inventory/aws/ec2.py
 
+[privilege_escalation]
+become = False
```

Comment:
Wanted to make sure we weren't escalating by default and playbooks could override it.  Actually it appears that false is the default anyway, so this can probably be dropped.